### PR TITLE
fix create app command so that the config sent in the request

### PR DESCRIPTION
**Pull request by albertshau**
comes from a config field in the file contents. This lets us
extend the object if we have more arguments in the future, and it
also makes it so that the same file can be used to create an app
through curl and the CLI

### CODE REVIEW

File: cdap-cli/src/main/java/co/cask/cdap/cli/command/app/CreateAppCommand.java [hunk 1]
```diff
@@ -67,7 +67,8 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     if (configPath != null) {
       File configFile = resolver.resolvePathToFile(configPath);
       try (FileReader reader = new FileReader(configFile)) {
-        config = GSON.fromJson(reader, JsonObject.class);
+        ConfigFile configContents = GSON.fromJson(reader, ConfigFile.class);
```
Comment (jwang47) [inline cdap-cli/src/main/java/co/cask/cdap/cli/command/app/CreateAppCommand.java]: Can we use whatever object the HTTP handler uses here instead?

Comment (albertshau) [inline cdap-cli/src/main/java/co/cask/cdap/cli/command/app/CreateAppCommand.java]: thats a good point, will do that

**Comment by albertshau** [comment]
@jwang47 addressed comment, could you take another look?

**Comment by jwang47** [comment]
LGTM

**Comment by albertshau** [comment]
http://builds.cask.co/browse/CDAP-RBT460

**Pull request merged**